liana/method/ml/estimations/_estimations.py

```diff
@@ -0,0 +1,100 @@
+from pandas import DataFrame
+from numpy import array, in1d
+from scipy.sparse import csr_matrix, lil_matrix
+from tqdm import tqdm
+from decoupler import run_ulm, run_mlm, run_wmean, run_wsum, run_mdt, run_udt, run_viper, run_ora, run_gsea, run_gsva
+
+
+def _metalinks_estimation(me_res, adata, verbose, est_fun = 'mean_per_cell', pass_mask = False, **kwargs) -> DataFrame: 
```

Comment:
Shouldn't this be a public fun that we use to connect to the other parts of liana - e.g. spatial?

Reply:
yes, but cant you just connect it internally ?